src/lib/y2partitioner/actions/create_partition_table.rb

```diff
@@ -77,7 +77,10 @@ def perform_action
       # @param type [Y2Storage::PartitionTables::Type]
       def create_partition_table(type)
         disk.remove_descendants
-        disk.create_partition_table(type)
+        ptable = disk.create_partition_table(type)
+        if ptable.type.is?(:msdos)
+          ptable.minimal_mbr_gap = Y2Storage::DiskSize.B(512)
```

Comment:
After talking with Ancor now I understood why we want to create a non optimal aligned mbr gap, but, why 512 B?

Reply:
When we are creating a partition without custom region option, does the resulting partition continue being optimal aligned at the beginning?

Reply:
0.5 KiB is the space used by the partition table itself. You cannot go below that.

As already said, this only brings us on par with gpt where this is already possible. This does not affect partition alignment in any way.

Reply:
I would appreciate if you append this explanation to the comment in the code.

Reply:
Our documentation of `PartitionTables::Msdos#mbr_gap` is not clear/precise enough, which makes those 512 B confusing. See https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/partition_tables/msdos.rb#L44

That states that mbr_gap is "_the space between the **end** of the MBR and the beginning of the first partition._". That is not true. Is the space between the START of the MBR (ie. the start of the disk) and the beginning of the first partition. Isn't it?

Reply:
Note that documentation is only at `Y2Storage` level. There is no such method in libstorage-ng. In the library there is only `minimal_mbr_gap` (which is not documented).

Reply:
> Is the space between the START of the MBR (ie. the start of the disk) and the beginning of the first partition. Isn't it?

Yes.